### DISCUSSION TITLE
Export notes / comments along with checkmarks

### DISCUSSION THREAD

**Discussion by hereismari** [iSoron/uhabits#1867] [Feature Requests]
Hi Loop community!

Now that notes and comments are allowed: https://github.com/iSoron/uhabits/discussions/194 (thanks for that, I love this feature!!)

Would be great to export the habits notes / comments of each habit along with its score / checked CSV data. 

Thanks!

**Comment by studgeek** [comment]
I believe this is a duplicate of #1567. So please vote/comment there also.
